Directory.Build.targets

```diff
@@ -255,6 +255,19 @@ Related: https://forum.unity.com/threads/6572-debugger-agent-unable-to-listen-on
 
     <Error Text="Smoke test failed with code $(SmokeTestExitCode)" Condition="'$(SmokeTestExitCode)' != '200'" />
     <Message Importance="High" Text="Smoke test PASSED" Condition="'$(SmokeTestExitCode)' == '200'" />
+
+    <!-- WIP
```

Comment:
This getting removed before we merge?

Reply:
Didn't manage to get it to work. 
I'll extract to a another branch if you don't want it here.

Reply:
updated